x-pack/test_serverless/functional/test_suites/search/index.ts

```diff
@@ -26,6 +26,7 @@ export default function ({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./rules/rule_details'));
     loadTestFile(require.resolve('./console_notebooks'));
     loadTestFile(require.resolve('./search_playground/playground_overview'));
+    loadTestFile(require.resolve('./search_synonyms/search_synonyms_overview'));
```

Comment:
This should be in the feature flagged index instead and you should enabled the ui setting in the `config.feature_flags.ts` file until the FF is removed.